### ISSUE TITLE
Unable to know what tests failed from report

### ISSUE THREAD

**Issue by kiran-redhat** [garris/BackstopJS#612]
When (if at all) some of the tests fails with some or other reason, report not showing those scenario names in report. It would be good if we can some how show what are these failed scenarios in the report?
**Summary:**
![image](https://user-images.githubusercontent.com/17115939/33684727-1c3a5434-db34-11e7-869c-bfb52e7ee740.png)
**Detailed:**
![image](https://user-images.githubusercontent.com/17115939/33684749-32238004-db34-11e7-8a2d-65d25aa7252c.png)

**Comment by garris** [comment]
@kiran-redhat You are correct -- these errors are not sent to the web UI -- however -- these errors are surfaced in the CLI report.   There is a new web UI in development on another branch.  We can look at adding the existing CLI errors/warnings to the new UI as well.

**Comment by alexus2005** [comment]
Not exactly on topic, but hope to help a bit. When this happened to me, I was Chromy error in CLI. Then I checked ports, and 9222, 9223 etc were used by another application on my machine. So after changing ports, it all works again.

**Comment by VladimirPittner** [comment]
@kiran-redhat meanwhile the new web UI is in developement (kudos @garris) you can click on expand the list of scenarios and from that you might guess which scenarios didnt run probably (if you have them indexed it will be even easier)

**Comment by kiran-redhat** [comment]
hey thanks, yep I saw that, also the easy way to figure out its at report
generation where to know for sure what failed.  I created this just
specific to html report
